pipeline.py

```diff
@@ -0,0 +1,31 @@
+"""The fuzzing main pipeline."""
+import argparse
+
+from result_classes import Result
+from stage.analysis_stage import AnalysisStage
+from stage.evaluation_stage import EvalationStage
+from stage.writing_stage import WritingStage
+
+
+class Pipeline():
+  """The fuzzing main pipeline, with 3 iterative stages."""
```

Comment:
can you add some more details (e.g. one sentence per stage) in the docstring?

Reply:
Yep, good point. Done!